redwood-treehouse-host/src/iosTest/kotlin/app/cash/redwood/treehouse/LayoutTester.kt

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (C) 2024 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.treehouse
+
+import app.cash.redwood.widget.UIViewChildren
+import platform.CoreGraphics.CGRectMake
+import platform.UIKit.UILayoutConstraintAxisVertical
+import platform.UIKit.UIStackView
+import platform.UIKit.UIStackViewAlignmentCenter
+import platform.UIKit.UIStackViewAlignmentFill
+import platform.UIKit.UIStackViewDistributionEqualSpacing
+import platform.UIKit.UIStackViewDistributionFillEqually
+import platform.UIKit.UIView
+
+/**
+ * This layout puts 3 children in a 120 x 120 vertical stackview.
+ *
+ * Each child has initial intrinsic dimensions of 60 x 20.
+ *
+ * ```
+ * .-------------------------.
```

Comment:
Big fan of this ascii visualization! I've even [made a GitHub pages app](https://seanho.github.io/ascii-layout/) to generate these.

Reply:
Ooooh fun!